contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -1140,6 +1140,11 @@ LANGUAGE C;
 
 ALTER PROCEDURE master_dbo.sp_dropserver OWNER TO sysadmin;
 
+CREATE OR REPLACE FUNCTION sys.OBJECT_NAME(IN object_id INT, IN database_id INT DEFAULT NULL)
```

Comment:
Here you have declared object_id as int and in C function you are doing
```
Oid 				object_id = PG_GETARG_OID(0);
```
Which makes it susceptible to overflow issues. Please keep the datatypes consistent and add negative test cases for overflow